src/test/java/com/eighthlight/fabrial/test/server/AppAcceptanceTest.java

```diff
@@ -0,0 +1,35 @@
+package com.eighthlight.fabrial.test.server;
+
+import com.eighthlight.fabrial.server.ServerConfig;
+import com.eighthlight.fabrial.test.TcpClient;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+@Tag("acceptance")
+public class AppAcceptanceTest {
+  Process appProcess;
+
+  @BeforeEach
+  void setUp() throws IOException {
+    // hard-coded to SNAPSHOT version. might need to fix this eventually...
```

Comment:
You may be able to pass the version info from gradle via system properties (`System.properties()`), but I'm not sure what gradle provides offhand.

Reply:
Yeah, I'll cross that bridge when I get to it 😉